CMakeLists.txt

```diff
@@ -69,8 +76,15 @@ if(VERBOSE)
 endif()
 
 # use -DEXTRA_INC="<path>;<path>" and -DEXTRA_LINK="<path>;<path>"
+set(EXTRA_INC "" CACHE STRING "Extra include directories separated by ;")
+set(EXTRA_LINK "" CACHE STRING "Extra link directories separated by ;")
```

Comment:
I wasn't able to pass these options (`EXTRA_INC` and `EXTRA_LINK`) on the command line until adding them here.  I'm not a CMake expert so feedback on ways to improve this are welcome.